website/docs/Installation/macos-install.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 3
+---
+
+# MacOS
+## Installing Podman Desktop on MacOS using .dmg file
+
+If you already have a Podman machine up and running, you can check out the [Downloads](/downloads/macos) section of this website to get the applicable .dmg file depending on your Mac Hardware Architecture (i.e. Intel or Apple M1).
+
+Simply download the file from the [Downloads](/downloads/macos) section and open it in your machine to install Podman Desktop.
+
```

Comment:
I would avoid `Machine` here to not confuse people with podman machine

Reply:
alright, will change it in the next push